src/components/Header/Header.tsx

```diff
@@ -13,24 +13,27 @@ export default function Header() {
   const pathname = usePathname();
   // we only change the color when on the home page
   // the osnap page is all light theme
-  const isNotHomePage = pathname?.split("#")[0] !== "/";
-  const isLightTheme = isNotHomePage || isLightThemeFromScroll;
+  const isHomePage = pathname?.split("#")[0] === "/";
+
+  const isLightTheme = !isHomePage || isLightThemeFromScroll;
+
+  const links = isHomePage ? homePageLinks : osnapPageLinks;
```

Comment:
on the home page we have links to the various sections. on the osnap page we have links back to the home page, and external links to the quickstart guide and the docs